src/plugins/displayLayout/CustomStringFormatterSpec.js

```diff
@@ -0,0 +1,83 @@
+import CustomStringFormatter from './CustomStringFormatter';
+import { createOpenMct, resetApplicationState } from 'utils/testing';
+
+const CUSTOM_FORMATS = [
+    {
+        key: 'sclk',
+        format: (value) => 2 * value
+    },
+    {
+        key: 'lts',
+        format: (value) => 3 * value
+    }
+];
+
+const valueMetadata = {
+    key: "sin",
+    name: "Sine",
+    unit: "Hz",
+    formatString: "%0.2f",
+    hints: {
+        range: 1,
+        priority: 3
+    },
+    source: "sin"
+};
+
+const datum = {
+    name: "1 Sine Wave Generator",
+    utc: 1603930354000,
+    yesterday: 1603843954000,
+    sin: 0.587785209686822,
+    cos: -0.8090170253297632
+};
+
+describe('CustomStringFormatter', function () {
+    let element;
+    let child;
+    let openmct;
+    let customStringFormatter;
+
+    beforeEach((done) => {
+        openmct = createOpenMct();
+        window.openmct = openmct;
```

Comment:
in [TelemetryAPI.prototype.addFormat](https://github.com/nasa/openmct/pull/3469/files#diff-76a745146d5fec5837d1366dab95d334a419d1ed72fce981d745dca1f2a43efbR463) function `this` was `window`

Reply:
Because of [line 48](https://github.com/nasa/openmct/pull/3469/files#diff-0a66fef9c225b9bc490f1c451b1699297eaf4ec167e11ef71ed872fa9028552dR48)? We should just use `.bind()` on line 48 to fix this rather than putting `openmct` on window.